third-party-audit/security-audit-2019/RFP.md

```diff
@@ -0,0 +1,118 @@
+# Request for Proposal
```

Comment:
@chetak123 Similarly, can you please rename this directory to `sig-security-external-audit` for consistency with the files' previous home in the k/community repo?